csrc/multidevice/executor.cpp

```diff
@@ -98,10 +98,29 @@ MultiDeviceExecutor::MultiDeviceExecutor(
       copyFusionAndChangeOutputs(completeFusion(), vals_to_allocate_);
 }
 
-void MultiDeviceExecutor::postKernel(SegmentedGroup* group) {
+void MultiDeviceExecutor::postKernel(
+    SegmentedGroup* group,
+    const LaunchParams& l_params) {
```

Comment:
```suggestion
    const LaunchParams& launch_params) {
```

The characters saved don't seem to be worth the confusion created for first-time readers.